### ISSUE TITLE
Magit and Gravatar support thoughts

### ISSUE THREAD

**Issue by grettke** [magit/magit#1418]
```
Good morning,

We've got a distributed team with a lot of members and platforms. The
project uses Git. Git client apps present range from the command line to
Magit to anything in the GUI. Recently one of the team members
introduced me to the Git client named Sourcetree [1]. What struck me as
valuable was the integration of Gravatar [2] pictures directly in the
commit browser. Initially I didn't think much of it, but now weeks later
it has really changed my perspective on commits simply by being able to
put a name to the face (to the commit). That got me curious about:

• How do do it in Magit
  • Is anyone already doing it?
  • Gnus has it [3]
• Would it be welcomed into Magit?
  • Or would it be an add-on?
• What are your thoughts?

Kind regards,

gcr



Footnotes
─────────
```
[1] [http://www.sourcetreeapp.com/]

[2] [https://en.gravatar.com/]

[3] [https://julien.danjou.info/blog/2010/gnus-gravatar-support]

**Comment by tarsius** [comment]
I have finally implemented this as part of #2216. This feature is also available from the `pu` branch and will likely be released in `v2.3.0`.

**Comment by grettke** [comment]
@tarsius Excellent! Users will *love* this.
